homeassistant/components/growatt_server/sensor_types/sensor_entity_description.py

```diff
@@ -0,0 +1,20 @@
+"""Sensor Entity Description for the Growatt integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorEntityDescription
+
+
+@dataclass
+class GrowattRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    api_key: str
+
+
+@dataclass
+class GrowattSensorEntityDescription(SensorEntityDescription, GrowattRequiredKeysMixin):
+    """Describes Growatt sensor entity."""
+
+    precision: int | None = None
```

Comment:
I made some changes in this class at #56806, you should add it here to not break `native_unit_of_measurement`